### PR TITLE
add inclusive layer testnet

### PR DESCRIPTION
**Pull request by aditya172926**
### Description

Add Inclusive Layer testnet chain

### Testing

Yes

### CODE REVIEW

**Comment by JunayedMh** [comment]
add the chain logo.svg file

**Comment by JunayedMh** [comment]
also add PR changeset file

**Comment by aditya172926** [comment]
Any updates here @JunayedMh .
Let me know if anything else is required.

File: chains/inclusive-layer/addresses.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+interchainAccountRouter: "0x36d152aBA87fF8F847CF9521fa23A27685F4F141"
+mailbox: "0x6103c366c286371a8c182619EaC9949e18106528"
+merkleTreeHook: "0x500E22f3327B19A20f33c5922C8965E47E511792"
+proxyAdmin: "0xe8b237a05e39945377CD55FB511CEAdF3c967e62"
+staticAggregationHookFactory: "0xA58A7d02Dc98E839f383401a303a813DE7B5B430"
+staticAggregationIsmFactory: "0x873aa4A0BADB5f3ED2189eA6696E2603da6384a1"
+staticMerkleRootMultisigIsmFactory: "0x10Bd789b9d66eFF07800Bf01dF9149aD5c8759A5"
+staticMessageIdMultisigIsmFactory: "0xcF7603362d0FC5a7cD17C1C43Bf35BC85a9415d7"
+testRecipient: "0x74Bdb3A9d82265d155A0b11d278d73Eaa2bE4C2b"
+validatorAnnounce: "0xAd0c03dfe7B8E2dF3fBc73f421CbfE0cC9de072e"
```
Comment (paulbalaji) [inline chains/inclusive-layer/addresses.yaml:12]:
```suggestion
validatorAnnounce: "0xAd0c03dfe7B8E2dF3fBc73f421CbfE0cC9de072e"

```

File: chains/inclusive-layer/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+domainId: 3461164656
+isTestnet: true
+name: inclusivelayertestnet
+nativeToken:
+  decimals: 18
+  name: Ether
+  symbol: ETH
+protocol: ethereum
+rpcUrls:
+  - http: http://192.241.245.190:18449
```
Comment (paulbalaji) [inline chains/inclusive-layer/metadata.yaml:23]:
```suggestion
  - http: http://192.241.245.190:18449

```

**Review by paulbalaji — COMMENTED** [review]
yarn lint:fix

File: .changeset/thick-mails-push.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': patch
```
Comment (JaunePomme) [inline .changeset/thick-mails-push.md:2]: Should be minor

File: chains/inclusive-layer/logo.svg
Comment (JaunePomme) [inline chains/inclusive-layer/logo.svg:1]: ran the minimifier https://jakearchibald.github.io/svgomg/

Comment (JaunePomme) [inline chains/inclusive-layer/logo.svg:1]: Run the minimifier https://jakearchibald.github.io/svgomg/ if you didn't

File: chains/inclusive-layer/metadata.yaml
Comment (JaunePomme) [inline chains/inclusive-layer/metadata.yaml:1]: -apiUrl missing,
-I believe the url of the blockExplorers isn't the right one, 
-May change the deployer to your own github as well like : 
deployer:
  name: aditya172926
  url: https://github.com/aditya172926

**Comment by JaunePomme** [comment]
Don't forget to run the prettier after all the changes

File: chains/inclusive-layer/metadata.yaml
Comment (aditya172926) [reply chains/inclusive-layer/metadata.yaml:1]: We are currently using that url for our block explorer.

File: chains/inclusive-layer/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - family: blockscout
+    name: Inclusive Layer Testnet Explorer
+    url: http://192.241.245.190
```
Comment (paulbalaji) [inline chains/inclusive-layer/metadata.yaml:5]:
```suggestion
  - apiUrl: http://192.241.245.190/api
  	family: blockscout
    name: Inclusive Layer Testnet Explorer
    url: http://192.241.245.190
```

**Review by paulbalaji — APPROVED** [review]

File: chains/inclusive-layer/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: http://192.241.245.190/api
+  	family: blockscout
+    name: Inclusive Layer Testnet Explorer
+    url: http://192.241.245.190
```
Comment (paulbalaji) [inline chains/inclusive-layer/metadata.yaml:6]:
```suggestion
  - apiUrl: http://192.241.245.190/api
    family: blockscout
    name: Inclusive Layer Testnet Explorer
    url: http://192.241.245.190
```

**Pull request merged**